### PR TITLE
Added changefeed message output to the sink page for examples

### PR DESCRIPTION
**Pull request by kathancox**
Fixes DOC-6225

Added examples of message output format for each of the sinks.

## Preview

https://deploy-preview-15848--cockroachdb-docs.netlify.app/docs/stable/changefeed-sinks.html#pub-sub-sink-messages

### CODE REVIEW

**Review by amruss — APPROVED** [review]
LGTM

**Comment by kathancox** [comment]
Hey @andf-crl, just a friendly reminder on this one in case it got lost in the holiday shuffle!

**Review by andf-crl — APPROVED** [review]
LGTM with one optional nit. Nice, thanks! Sorry for the delay!

File: v22.2/changefeed-messages.md [hunk 1]
```diff
@@ -245,6 +245,25 @@ You can use the [`format=csv`](create-changefeed.html#format) option to emit CSV
 
 See [Export Data with Changefeeds](export-data-with-changefeeds.html) for detail on using changefeeds to export data from CockroachDB.
 
+~~~
```
[the remark this answers is not in the document]
Comment (kathancox) [reply v22.2/changefeed-messages.md:250]: Added! Thanks

**Pull request merged**